client/src/components/pages/reports/tests/Reports_test.js

```diff
@@ -0,0 +1,230 @@
+import React from 'react';
```

Comment:
the name of this file should be Reports.test.js, and it is also testing the register page.

Reply:
mistake random file left from a branch i was messing around on a while ago, removed